### DISCUSSION TITLE
BurntToast Notification not showing when GPO setting applied

### DISCUSSION THREAD

**Discussion by MathFlyAIBV** [Windos/BurntToast#158] [General]
Hi There,

First I want to say that I'm super pleased with the BurntToast module, that work great and does exactly what I want.

I'm facing a very weird issue that I cannot understand at the moment.
In my organization, I've a GPO that enable the setting "Always wait for the network at computer startup and logon". so that the computer waits for network initialisation to log the user in.

So far it gave no issues. 

But now, We do have some computer running Windows 10 20H2 x64. and it seems that when this setting is active, the notifications do not show.
There are no errors, no exception messages, it just do nothing.
I've tested all the applied GPOs one by one and this setting seems to be responsible.
Also, when I enable it on the local policies, it has the same behavior.

I'm not sure what's going on with the module and I cannot seem to link the issue to the setting, so I was wondering if maybe you might have an idea of what's happening ?

Thanks !

MathFly_

**Comment by MathFlyAIBV** [comment]
Just to add, I use a very simple notification, here is the code :
`New-BurntToastNotification -Button @(
    New-BTButton -Content "AIBV Form" -Arguments "https://www.aibv.be/objectif-doelstelling"
) -Text "Don't Forget", "Fill in the Form"`

**Comment by Windos** [comment]
That is **very** odd. There's nothing about that GPO setting that should have anything to do with notifications.

How is the toast being triggered? Is it from a GPO in and of itself (login script) or are you opening up a console and doing it manually?

**Reply by MathFlyAIBV** [reply]
Yeah, I know !
I'm pulling my hair on that for two days now.

The command is launched in a PowerShell script that is called via a VBS script (To hide the very quick PowerShell window that appear) triggered by a Scheduled task running every hour.

But I also made some test and ran the command directly from a PowerShell prompt, and the command will go through, won't give error, but nothing will happen.

I found out that it's not limited to this build of Windows, but previous as well.
It's definitely profile related as we're using the Toast only for one shared user that has a roaming profile and it works on other accounts without Roaming profile or local account.

I've tried to go deep in the event viewer but was not able to find something relevant to this.
I see that the toast is indeed sent to the the system in the logs : Microsoft -> Windows -> PushNotifications-Platform : 
Toast with notification tracking id 26711 is being delivered to {1AC14E77-02E7-4E5D-B744-2EB1AE5198B7}\WindowsPowerShell\v1.0\powershell.exe on session 1.
But still nothing comes.

The weird thing is that if I run stuff like what is explained on that page : 
https://byteben.com/bb/deploy-service-announcement-toast-notifications-in-windows-10-with-memcm/

I can see the notification, but the title seems to be somehow broken as it should display "Windows PowerShell".
So it might be related... but not sure...

So don't break your head too much on that, I'll keep digging on this issue, it was just to know if you had a track to follow or any small idea depending on how this works. 

Still the module is great :)

**Reply by MathFlyAIBV** [reply]
![image](https://user-images.githubusercontent.com/80772526/111603222-6c04c300-87d4-11eb-8471-eb6d5808309a.png)
Here's the "Broken" title I was mentioning ..

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
